wowup-electron/src/app/components/options/options-addon-section/options-addon-section.component.ts

```diff
@@ -40,8 +40,8 @@ export class OptionsAddonSectionComponent implements OnInit, OnDestroy {
         takeUntil(this.destroy$),
         debounceTime(300),
         switchMap((ch) => {
-          if (ch.cfV2ApiKey) {
-            return from(this._preferenceStorageService.setAsync(PREF_CF2_API_KEY, ch.cfV2ApiKey));
+          if ('cfV2ApiKey' in ch) {
```

Comment:
This change will break it, by allowing the set to be passed undefined, which breaks things downstream.

Reply:
What about this?
```suggestion
          if ('cfV2ApiKey' in ch && typeof ch.cfV2ApiKey === "string") {
```

Reply:
You could also say `if( typeof ch?.cfV2ApiKey === 'string')`

Reply:
Yeah that's probably better 😂